.circleci/config.yml

```diff
@@ -7,8 +7,172 @@ orbs:
 
 
 jobs: 
+   build-and-push-docker-image:
+    environment:
+      - IMAGE: taraxa-node
+    machine:
+      image: ubuntu-1604:202007-01
+      docker_layer_caching: true
+    resource_class: xlarge
+    steps:
+      - checkout
+      - run: 
+         name: Checkout Submodules
+         command: |
+              git submodule sync
+              git submodule update --init --recursive --jobs 8
+      - run:
+         name: Prepare Environment
+         command: |
+           if [[ ${CIRCLE_SHELL_ENV} =~ "localbuild" ]]; then
+              export CIRCLE_BRANCH=circleci
+              export CIRCLE_BUILD_NUM=300
+           fi
+           echo "export HELM_TEST_NAME=$(echo $CIRCLE_BRANCH | sed 's/[^A-Za-z0-9\\-]*//g' | tr '[:upper:]' '[:lower:]')" >>$BASH_ENV
+           echo "export DOCKER_BRANCH_TAG=$(./scripts/docker_tag_from_branch.sh $CIRCLE_BRANCH)" >>$BASH_ENV
+           echo "export START_TIME=$(date +%Y%m%d-%Hh%Mm%Ss)" >>$BASH_ENV
+           echo "export GCP_IMAGE=gcr.io/${GOOGLE_PROJECT_ID}/${IMAGE}" >> $BASH_ENV
+           sudo service apport stop
+           sudo bash -c 'echo /tmp/core.%t.%e.%p >/proc/sys/kernel/core_pattern'
+
+           mkdir -p  $PWD/tmp_docker 
```

Comment:
I think this only works because we are using executer type machine
if we don't plan to move to execute docker, then this hack should be ok, but I would rather try to use docker.